docpages/example_programs/interactions_and_components/private-messaging.md

```diff
@@ -0,0 +1,80 @@
+\page private-messaging Sending private messages
+
+Sometimes it's simply not enough to ping someone in a server with a message, and we get that. That's why you can private message people! This tutorial will cover how to make a command that will either message the author of the command or message a specified user! This is done with a command option! So, let's get into it!
+
+\note This tutorial makes use of callbacks. For more information about that, visit \ref callback-functions "Using Callback Functions".
+
+~~~~~~~~~~{.cpp}
+#include <dpp/dpp.h>
+
+int main()
+{
+    /* Create the bot */
+    dpp::cluster bot("token");
+
+    bot.on_log(dpp::utility::cout_logger());
+
+    /* The event is fired when someone issues your commands */
+    bot.on_slashcommand([&bot](const dpp::slashcommand_t & event) {
+
+        /* Check which command they ran */
+        if (event.command.get_command_name() == "pm") {
+
+            /* If there was no input, we want to only send a message to the command author. */
+            if(event.get_parameter("user").index() == 0) {
```

Comment:
There is no space between `if` and `(` but there is in other spaces

Reply:
This is because the other parts are copy-pasted from the other example pages. The parts where there is no space comes from my IDE. Will be fixed.

Reply:
No worries 👍